### ISSUE TITLE
SKAdNetwork support

### ISSUE THREAD

**Issue by egorlitvinenko** [mopub/mopub-ios-sdk#327]
Do you have plans to add support for SKAdNetwork into SDK?

- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### Device model and OS Version:
IOS 14

#### Xcode Version:
Xcode 12

**Comment by chauduyphanvu** [comment]
@egorlitvinenko, thanks for getting in touch! 

Regarding SKAdNetwork, MoPub is exploring whether to support it. We do not have a timeline at this point. Let us know if there are any questions!

Vu Chau
MoPub Team

**Comment by egorlitvinenko** [comment]
@chauduyphanvu 
What is the reason you have closed the ticket?
Any updates on it?

**Comment by chauduyphanvu** [comment]
@egorlitvinenko, we currently do not have an update yet, but will let you know!

**Comment by egorafanasenko** [comment]
@chauduyphanvu could you give us any estimation on this. As a game developer that is using **mopub** we need to understand whether you are going to support SKAdNetwork, or we need to do it on our side. thanks!

**Comment by chauduyphanvu** [comment]
@egorafanasenko, yes, we will be supporting SKAdNetwork. More details will come soon—stay tuned!

**Comment by egorlitvinenko** [comment]
https://developers.mopub.com/dsps/integration/ios14proposal/
